signhost.php

```diff
@@ -293,30 +285,157 @@ function __construct($type) {
 
 }
 
-class iDEAL extends Verification {
+class IdealVerification extends Verification implements JsonSerializable {
 
     public $Iban; // String
```

Comment:
Maybe document the properties with @var syntax?
https://stackoverflow.com/questions/2194822/how-to-document-class-properties-in-php-5-with-phpdocumentor#2194835
Don't if there are also tags for readonly and optional